src/v/test_utils/boost_result_redirect.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include <boost/test/detail/global_typedef.hpp>
+#include <boost/test/unit_test_log.hpp>
+#include <boost/test/unit_test_suite.hpp>
+
+#include <fstream>
+#include <iomanip>
+
+/**
+ * This test configuration object is used to redirect the JUNIT result output
+ * (if enabled) to the location specified by the XML_OUTPUT_FILE environment
+ * variable. This variable is set by the bazel test runner so that the test
+ * writes the results in a local that bazel can subsequently consume.
+ * Ref: CORE-8013.
+ */
+struct BazelBoostResultHandler {
```

Comment:
Should this be snake case? I don’t really care…

Reply:
Yes, of course, sorry (I inherited this case from some sample code I based this on).

Reply:
Fixed in push https://github.com/redpanda-data/redpanda/commit/05d64a5f1baeeff17522ee7bf466f992412edf66.